shark/shark_runner.py

```diff
@@ -48,10 +48,16 @@ def __init__(
         if self.frontend in ["pytorch", "torch"]:
             self.model = get_torch_mlir_module(self.model, input, dynamic,
                                                jit_trace, from_aot)
-        (
+            (
             self.iree_compilation_module,
             self.iree_config,
-        ) = get_iree_compiled_module(self.model, device)
+            ) = get_iree_compiled_module(self.model, device)
+
+        if self.frontend in ["tensorflow", "tf", "mhlo"]:
+            (
+            self.iree_compilation_module,
+            self.iree_config,
+            ) = get_iree_compiled_module(self.model, device, self.frontend)
```

Comment:
Can be combined into just get_iree_compiled_module(self.model, device, self.frontend)

Reply:
Actually, it's fine for now, I need to also push up a change that has conflict with this, will clean up there :)